simvue/sender.py

```diff
@@ -37,11 +37,20 @@
 _logger = logging.getLogger(__name__)
 
 
+def _log_upload_failed(file_path):
```

Comment:
Please add basic docstring, doesnt necessarily need arguments but should summarise. Also type function.